CMSgov/mint-app
# [task: EASI-3923] General Characteristics And Basics Readonly Updates

# EASI-3923


[Figma](https://www.figma.com/file/6tkIhfNfcsKEAkqXo1qvXH/MINT---Read-Only-View?type=design&node-id=4386-18136&mode=design&t=LxID0gXBvf1c7xnl-0)

## Changes and Description

- Updated General Characteristics and Model Basics readonly to use new translation configurations
- Added support for readonly translation options
- Added additional extended type for questions that are both a parent and child
- Added support for rendering lists conditionally under radio values
- Added support for adding single tooltip values on radio



## How to test this change

Verify General Characteristics readonly matches Figma
Verify Model Basics readonly behaves exactly as it did before (No figma changes)

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
